Formula/f/foot.rb

```diff
@@ -0,0 +1,62 @@
+class Foot < Formula
+  desc "Fast, lightweight and minimalistic Wayland terminal emulator"
+  homepage "https://codeberg.org/dnkl/foot"
+  url "https://codeberg.org/dnkl/foot/archive/1.18.1.tar.gz"
+  sha256 "59d22187f7ceaaaa570a5299b102e8f4692826e98785f89ad9d8911802ccc000"
+  license "MIT"
+
+  depends_on "cmake" => :build
+  depends_on "meson" => :build
+  depends_on "ninja" => :build
+  depends_on "pkg-config" => :build
+  depends_on "scdoc" => :build
+
+  depends_on "fontconfig"
+  depends_on "freetype"
+  depends_on "libxkbcommon"
+  depends_on :linux
+  depends_on "pixman"
+  depends_on "wayland"
+
+  def install
+    ENV["CFLAGS"] = "#{ENV["CFLAGS"]} -O3"
+
+    mkdir_p "bld/release"
+
+    cd "bld/release" do
+      system "meson",
+        "setup",
+        "-Dterminfo-base-name=foot-extra",
+        "--buildtype=release",
+        "--prefix=#{prefix}",
+        "../.."
+      system "ninja"
+      system "ninja", "test"
+      system "ninja", "install"
+    end
+  end
+
+  test do
+    config_dir = testpath/".config/foot"
+    config_file = config_dir/"foot.ini"
+
+    mkdir_p config_dir
+
+    File.write config_file, <<-EOF
+      [cursor]
+      style=blok
+    EOF
+
+    assert_match(
+      /blok: not one of 'block', 'underline', 'beam'/,
+      `#{bin}/foot --check-config 2>&1`,
```

Comment:
```suggestion
      shell_output("#{bin}/foot --check-config 2>&1"),
```

Reply:
Changed, with a small tweak for the return value.